src/simulation/m_time_steppers.fpp

```diff
@@ -928,9 +892,27 @@ contains
             idwint, &
             gm_alpha_qp%vf)
 
```

Comment:
I get the feeling that something in here could be where Frontier is going wrong.